lib/workflow/client.js

```diff
@@ -103,6 +103,13 @@ class Workflow {
             meta
           })
         });
+      },
+      extend: ({ meta: comment }) => {
+        this.validate({ comment, taskId }, 'comment', 'taskId');
+        return this.client(`/${taskId}/extend`, {
```

Comment:
This doesn't feel right. We should be updating the task directly via `PUT /:taskId` rather than adding new endpoints specifically to update particular fields.